canvas_cli/main.py

```diff
@@ -1,12 +1,19 @@
 import importlib.metadata
+import json
+import random
 from pathlib import Path
 from typing import Optional
 
+import grpc
 import typer
+from typing_extensions import Annotated
```

Comment:
I think `typing_extensions` might befor backward compatibility, but `Annotated` lives in `typing` after 3.11.

```suggestion
from typing import Annotated
```